checker/jdk/nullness/src/javax/xml/parsers/FactoryConfigurationError.java

```diff
@@ -0,0 +1,137 @@
+/*
+ * Copyright (c) 2000, 2005, Oracle and/or its affiliates. All rights reserved.
+ * ORACLE PROPRIETARY/CONFIDENTIAL. Use is subject to license terms.
+ *
+ *
+ *
+ *
+ *
+ *
+ *
+ *
+ *
+ *
+ *
+ *
+ *
+ *
+ *
+ *
+ *
+ *
+ *
+ *
+ */
+
+package javax.xml.parsers;
+
+import org.checkerframework.checker.nullness.qual.Nullable ;
+
+/**
+ * Thrown when a problem with configuration with the Parser Factories
+ * exists. This error will typically be thrown when the class of a
+ * parser factory specified in the system properties cannot be found
+ * or instantiated.
+ *
+ * @author <a href="mailto:Jeff.Suttor@Sun.com">Jeff Suttor</a>
+ * @version $Revision: 1.7 $, $Date: 2010-11-01 04:36:09 $
+ */
+
+public class FactoryConfigurationError extends Error {
+    private static final long serialVersionUID = -827108682472263355L;
+
+    /**
+     *<code>Exception</code> that represents the error.
+     */
+    private Exception exception;
```

Comment:
It seems that this needs `@Nullable`.  That would be more accurate, and would also enable you to remove the `@SuppressWarnings("nullness")` on [line 52](https://github.com/typetools/checker-framework/pull/1808/files#diff-b3b560eba2d73854dde827a8969a4e3eR52). and line 64.